client/tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      colors: {
+        "main-purple": "#5556B8",
```

Comment:
can you generate the full-color variations please?
use this tool https://javisperez.github.io/tailwindcolorshades/